### PR TITLE
update version

### PR DESCRIPTION
**Pull request by diarmuidmcg**

### CODE REVIEW

**Comment by diarmuidmcg** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/miamtech/MiamCoursesUUI/56?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#56** <a href="https://app.graphite.dev/github/pr/miamtech/MiamCoursesUUI/56?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 10  dependent PRs ([#57](https://github.com/miamtech/MiamCoursesUUI/pull/57) <a href="https://app.graphite.dev/github/pr/miamtech/MiamCoursesUUI/57?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#62](https://github.com/miamtech/MiamCoursesUUI/pull/62) <a href="https://app.graphite.dev/github/pr/miamtech/MiamCoursesUUI/62?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#63](https://github.com/miamtech/MiamCoursesUUI/pull/63) <a href="https://app.graphite.dev/github/pr/miamtech/MiamCoursesUUI/63?utm_source=stack-comment-icon" target="_blank">Graphite</a> and 7 others) 👈 <a href="https://app.graphite.dev/github/pr/miamtech/MiamCoursesUUI/56?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#54** <a href="https://app.graphite.dev/github/pr/miamtech/MiamCoursesUUI/54?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#73](https://github.com/miamtech/MiamCoursesUUI/pull/73) <a href="https://app.graphite.dev/github/pr/miamtech/MiamCoursesUUI/73?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: .idea/deploymentTargetSelector.xml
Comment (ThibaultMiam) [inline .idea/deploymentTargetSelector.xml:1]: dont push this file

File: .idea/gradle.xml
Comment (ThibaultMiam) [inline .idea/gradle.xml:1]: same

File: .idea/kotlinc.xml
Comment (ThibaultMiam) [inline .idea/kotlinc.xml:1]: same

File: CoursesUUI/src/main/java/tech/miam/coursesuui/template/myMeal/CoursesUMyMealButton.kt [hunk 1]
```diff
@@ -51,24 +44,17 @@ class CoursesUMyMealButton: MyMealButtonSuccess {
                     horizontalArrangement = Arrangement.SpaceBetween,
                     verticalAlignment = Alignment.CenterVertically,
                     modifier = Modifier
-                        .clickable { params.onClick() }
-                        .padding(12.dp)
+                        .padding(start = 16.dp)
```
Comment (ThibaultMiam) [inline CoursesUUI/src/main/java/tech/miam/coursesuui/template/myMeal/CoursesUMyMealButton.kt]: no longer onclick action on a button ?

File: app/build.gradle [hunk 2]
```diff
@@ -30,12 +30,12 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
         debug {
-            firebaseAppDistribution {
-                artifactType = "APK"
-                releaseNotes= "test Courses U test app"
-                testers = "thomas.huyghues@miam.tech"
-                serviceCredentialsFile = "miamandroid-14845a0ac147.json"
-            }
+//            firebaseAppDistribution {
```
Comment (ThibaultMiam) [inline app/build.gradle]: should we remove it ?

File: app/src/main/java/tech/miam/coursesUDemoApp/features/miam/template/MiamManager.kt [hunk 3]
```diff
@@ -19,13 +21,15 @@ import kotlinx.coroutines.withContext
...
 import tech.miam.coursesUDemoApp.basket.BasketEvent
+import tech.miam.coursesUDemoApp.data.models.Attributes
 import tech.miam.coursesUDemoApp.data.models.Product
 import tech.miam.coursesUDemoApp.features.miam.template.MiamSdkHelper.pushProductToRetailer
 import tech.miam.coursesUDemoApp.features.products.ProductsRepository
 import tech.miam.coursesuui.config.MiamTemplateManager
 import timber.log.Timber
 
-object MiamSdkHelper : CoroutineScope by CoroutineScope(Dispatchers.Main), KoinComponent, BasketPublisher, BasketSubscriber {
+object MiamSdkHelper : CoroutineScope by CoroutineScope(Dispatchers.Main), KoinComponent, BasketPublisher,
```
Comment (ThibaultMiam) [inline app/src/main/java/tech/miam/coursesUDemoApp/features/miam/template/MiamManager.kt]: still using koin ?

File: app/build.gradle (hunk 2 above)
Comment (diarmuidmcg) [reply app/build.gradle]: yes, done

File: app/src/main/java/tech/miam/coursesUDemoApp/features/miam/template/MiamManager.kt (hunk 3 above)
Comment (diarmuidmcg) [reply app/src/main/java/tech/miam/coursesUDemoApp/features/miam/template/MiamManager.kt]: removed it from this file (still used in other files)

File: CoursesUUI/src/main/java/tech/miam/coursesuui/template/myMeal/CoursesUMyMealEmpty.kt [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+                verticalArrangement = Arrangement.Center
+            ) {
+                Image(
+                    painter = painterResource(R.drawable.empty_white),
+                    contentDescription = "empty page",
+                    Modifier.padding(top = 16.dp, bottom = 24.dp),
+                    colorFilter = ColorFilter.tint(Color.White)
+                )
+                Text(
+                    text = "Vous n'avez aucune idées repas",
```
Comment (antonin-francois-mealz) [inline CoursesUUI/src/main/java/tech/miam/coursesuui/template/myMeal/CoursesUMyMealEmpty.kt:60]: no localisation ?

**Pull request merged**